dell/latitude/9430/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+  ];
+
+  boot.kernelParams = [
+    # See https://discourse.nixos.org/t/i915-driver-has-bug-for-iris-xe-graphics/25006/12
+    # jheidbrink reports that without this setting there is a very high lag in Sway which makes it unusable
+    "i915.enable_psr=0"
```

Comment:
Is this still necessary if we set this?

```
hardware.intelgpu.driver = "xe";
```

Reply:
I tested with that config (and Kernel 6.8 as it is required for "xe") with `i915.enable_psr=0` commented out, and the lag appeared again. Setting `enable_psr=0` again made it disappear again.